src/v2/Components/ArtworkFilter/ArtworkFilters/FilterExpandable.tsx

```diff
@@ -1,11 +1,51 @@
-import { Expandable, ExpandableProps, useThemeConfig } from "@artsy/palette"
-import React from "react"
+import {
+  Box,
+  Expandable,
+  ExpandableProps,
+  useThemeConfig,
+} from "@artsy/palette"
+import React, { useContext, useEffect, useRef, useState } from "react"
+import { getElementParams } from "./helpers"
+import { ScrollRefContext } from "./useScrollContext"
 
 export const FilterExpandable: React.FC<ExpandableProps> = props => {
   const tokens = useThemeConfig({
     v2: { mb: 1 },
     v3: { mb: 6 },
   })
 
-  return <Expandable mb={tokens.mb} {...props} />
+  // @ts-expect-error STRICT_NULL_CHECK
```

Comment:
This will probably error out in our CI as we can remove these, but we can't add any additional one's. 

(This is part of some tooling we added to migrate to strick type checking in force.)